### ISSUE TITLE
[2.0.0] Form element clear do not works as in 1.3.4

### ISSUE THREAD

**Issue by DestinyMKas** [phalcon/cphalcon#10044]
Hello,

1.3.4 version had form with password field. I don't want to render password, so I used $element->clear(). In 2.0.0 it not works.
I tried to do $element->setDefault('')->clear(), but it didn't worked too.

As a workaround I did:
$this->getEntity()->password = '';
But this method has some issues.

**Comment by DestinyMKas** [comment]
#10069 pull request initiated

**Comment by DestinyMKas** [comment]
Wrong comment

**Comment by sergeyklay** [comment]
Fixed in `3.0.x` branch.
Could you please check `3.0.x` branch?

```sh
git clone git@github.com:phalcon/cphalcon.git
cd cphalcon
git checkout 3.0.x

zephir fullclean
zephir build
```
